simvue/api/objects/alert/fetch.py

```diff
@@ -28,6 +28,14 @@ class Alert:
     def __new__(cls, identifier: str, **kwargs) -> AlertType:
         """Retrieve an object representing an alert either locally or on the server by id"""
         _alert_pre = AlertBase(identifier=identifier, **kwargs)
+        if (
+            identifier is not None
+            and identifier.startswith("offline_")
+            and not _alert_pre._staging.get("source", None)
+        ):
+            raise RuntimeError(
+                "Cannot determine Alert type - this is likely because you are attempting to reconnect to an offline alert which has already been sent to the server. To fix this, use the exact Alert type instead (eg MetricThresholdAlert, MetricRangeAlert etc)."
```

Comment:
Split across multiple lines to meet 78 character limit.

Not for you to implement but wondering if we should add:

```python
@property
def staging(self) -> dict[str, typing.Any]:
    return self._staging
```

to  `base.py